src/DynamoPackages/PackageManagerSearchElement.cs

```diff
@@ -42,6 +42,8 @@ public int Votes
         public int UsedBy { get { return this.Header.used_by.Count; } }
         public string LatestVersion { get { return Header.versions[Header.versions.Count - 1].version; } }
         public string LatestVersionCreated { get { return Header.versions[Header.versions.Count - 1].created; } }
+        public string SelectedVersion { get; set; }
```

Comment:
theoretically this should be on view model layer but I guess such layer does not exist for search element...

Reply:
Hmm .. no, not really :/ Of course, you are right, we shouldn't be binding directly to the Model. Strange, the existing bindings go straight to the model. Should I refactor those as well? Or not touch the existing ones.

Reply:
hmm, we can leave that as a ToDo in a new PR if that is OK

Reply:
Moved the `SelectedVersion` to the VM, we will fix the rest in a separate PR as you said, @QilongTang !